CONTRIBUTING.md

```diff
@@ -54,5 +54,23 @@ accept your pull requests.
 1.  Lint (and maybe fix) any changes:
 
         npm run fix
-
-[node]: https://nodejs.org/en/
\ No newline at end of file
+        
+## Testing a new feature using CLI
+
+1. After you've written some new code, in order to test it out, you can use the [CLI][CLI]. 
+   
+   For example, the CLI command:
+   
+   `release-please release-pr 
```

Comment:
@ddixit14 Is there any step to create the "release-please" command?

Reply:
No, no step to create the command.

Reply:
Then how to create the `release-please` command from the source?